andela/AndelaEatsUI
# #161079274 Implement add engagement functionality

#### What does this PR do?
This PR implements add engagement functionality.

#### Description of Task to be completed?
Add engagement functionality

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-admin-create-engagement-functunality-161079274`
- run these commands `npm run bundle` and `npm run dev` 
- Log in as an `Admin`, navigate to `admin dashboard`, select `Engagement Icon`, and click on `Add Engagement`
- run the command `npm run test-dev` to ensure all tests are passing

#### What are the relevant pivotal tracker stories?
- [#161079274](https://www.pivotaltracker.com/n/projects/2178749/stories/161079274)

#### Background Context
Nil

#### Screenshots (If Applicable)
<img width="1276" alt="screen shot 2018-11-15 at 1 41 48 pm" src="https://user-images.githubusercontent.com/21138774/48553722-9da3b180-e8dc-11e8-9448-e9337d3c3e1f.png">
